app/models/provider.rb

```diff
@@ -1,11 +1,14 @@
 class Provider
   include ActiveModel::Model
+  include Rails.application.routes.url_helpers
 
-  attr_accessor :slug, :name
+  attr_accessor :slug, :name, :link_external
```

Comment:
```suggestion
  attr_accessor :slug, :name, :external_link
```
`link_external` c'est pas anglais :D